tests/test_request_cache.py

```diff
@@ -57,6 +57,39 @@ def foo(a, b, c, x=None, y=None, z=None):
     )
 
 
+@pytest.mark.parametrize('cache_set_call, expected_redis_call', (
```

Comment:
Optional: I feel the param variable names might be a bit more readable as something like `requested_cache_ttl, expected_redis_cache_ttl`. I guess I was expecting to see the word `ttl` in the variable names